clickhouse/tests/docker/init.sql

```diff
@@ -1,3 +1,7 @@
+-- Attempts to fix flaky tests
+DROP TABLE IF EXISTS tableau;
+DROP TABLE IF EXISTS tableau_distributed;
+--
 CREATE TABLE tableau (id Int64) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{layer}-{shard}/tableau', '{replica}') PARTITION BY id ORDER BY id;
```

Comment:
Do you think this is better:
`CREATE TABLE IF NOT EXISTS tableau (id Int64) ENGINE = ReplicatedMergeTree('/clickhouse/tables/{layer}-{shard}/tableau', '{replica}') PARTITION BY id ORDER BY id;`
?

Reply:
I thought about that but I wanted to be sure that we are not going to insert the same data twice just in case in the table

Reply:
Not sure to understand you

Reply:
After our call:

- We see that each time the CI fails, we have this error on only 5 of the 6 clickhouse containers
- We think that the table is replicated from one instance to the other, so we do not drop them
- Replication configuration: https://github.com/DataDog/integrations-core/blob/8375f36e5182763100a1162e4fc277f29f34be9e/clickhouse/tests/docker/override_configs/remote_servers.xml#L2
- Doc about replication: https://clickhouse.com/docs/en/engines/table-engines/mergetree-family/replication/

Reply:
So let's go with the `CREATE TABLE IF NOT EXISTS`. Thanks!